mirar/references/sdss.py

```diff
@@ -31,12 +33,12 @@ def _get_reference(self, image: Image) -> (fits.PrimaryHDU, fits.PrimaryHDU):
 
         wcs = WCS(header)
 
-        ra_cent, dec_cent = wcs.all_pix2world(nx, ny, 0)
+        ra_cent, dec_cent = wcs.all_pix2world(nx / 2, ny / 2, 0)
 
-        if not in_sdss(ra_deg=ra_cent, dec_deg=dec_cent):
-            err = "Image does not overlap SDSS"
-            logger.error(err)
-            raise NotInSDSSError(err)
+        # if not in_sdss(ra_deg=ra_cent, dec_deg=dec_cent):
```

Comment:
why is this commented out? Is it not needed anymore? Just delete if so

Reply:
https://github.com/winter-telescope/mirar/issues/719